server_apps/data_transfer/Downloads/DownloadFiles.sql

```diff
@@ -2275,95 +2275,126 @@ drop view zfinpubs;
-select feature_zdb_id, szm_term_ont_id, feature_abbrev, feature_name, ftrtype_type_display, featassay_mutagen,featassay_mutagee, construct.mrkr_zdb_id, construct.mrkr_name, createdby.fmrel_mrkr_zdb_id, str.mrkr_name
-from feature, feature_type, feature_assay, so_zfin_mapping, feature_marker_relationship cst, marker construct, feature_marker_relationship createdby, marker str
-where feature_type =ftrtype_name
- and szm_object_type = feature_type
- and feature_zdb_id = featassay_feature_zdb_id
- and feature_zdb_id = cst.fmrel_ftr_zdb_id
- and cst.fmrel_mrkr_zdb_id = construct.mrkr_zdb_id
- and construct.mrkr_type  in ('TGCONSTRCT','GTCONSTRCT','PTCONSTRCT','ETCONSTRCT')
-and feature_Type in ('TRANSGENIC_INSERTION')
-and cst.fmrel_type = 'is allele of'
-and featassay_mutagen in ('TALEN', 'CRISPR', 'DNA and TALEN', 'DNA and CRISPR')
-and createdby.fmrel_type = 'created by'
-and feature_zdb_id = createdby.fmrel_ftr_zdb_id
-and str.mrkr_zdb_id = createdby.fmrel_mrkr_zdb_id
+-- step 6
+-- Gives an empty set as of 11/12/2025.
+-- Also, logic overlaps with step 4 -- step4 having the condition cst.fmrel_type != 'is allele of' and step 6 having cst.fmrel_type = 'is allele of'.
+-- See comments for step 5 about this.
+insert into tmp_features (feature_id, term_o_id, f_abbrev, f_name, ftypedisp, mutagen, mutagee, construct_id, construct_name, created_by_zdb_id, created_by_name, step)
+select feature_zdb_id, szm_term_ont_id, feature_abbrev, feature_name, ftrtype_type_display, featassay_mutagen,featassay_mutagee, construct.mrkr_zdb_id, construct.mrkr_name, createdby.fmrel_mrkr_zdb_id, str.mrkr_name, 'step6'
+from
+    feature join feature_type on feature_type =ftrtype_name
+            join feature_assay on feature_zdb_id = featassay_feature_zdb_id
+            join so_zfin_mapping on szm_object_type = feature_type
+            join feature_marker_relationship cst on feature_zdb_id = cst.fmrel_ftr_zdb_id
+            join marker construct on construct.mrkr_zdb_id = cst.fmrel_mrkr_zdb_id
+            join feature_marker_relationship createdby on feature_zdb_id = createdby.fmrel_ftr_zdb_id
+            join marker str on str.mrkr_zdb_id = createdby.fmrel_mrkr_zdb_id
+where
+    construct.mrkr_type in ('TGCONSTRCT','GTCONSTRCT','PTCONSTRCT','ETCONSTRCT')
+  and feature_Type in ('TRANSGENIC_INSERTION')
+  and cst.fmrel_type = 'is allele of'
+  and featassay_mutagen in ('TALEN', 'CRISPR', 'DNA and TALEN', 'DNA and CRISPR')
+  and createdby.fmrel_type = 'created by'
 order by lower(feature_abbrev);
 
+-- step 7 (Add any features that are not already included)
+insert into tmp_features (feature_id, term_o_id, f_abbrev, f_name, ftypedisp, mutagen, mutagee, step)
+select feature_zdb_id, szm_term_ont_id, feature_abbrev, feature_name, ftrtype_type_display, featassay_mutagen,featassay_mutagee, 'step7'
+from
+    feature join feature_type on feature_type = ftrtype_name
```

Comment:
Is it possible these joins are causing some features to be excluded?

Reply:
> Is it possible these joins are causing some features to be excluded?

I double checked and there are 37 more features that are excluded by the inner join. I have modified the PR to include those:

[Features-Remaining.xlsx](https://github.com/user-attachments/files/23592313/Features-Remaining.xlsx)